docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -54,20 +52,12 @@ echo "192.168.121.9	fed-master
 192.168.121.65	fed-node" >> /etc/hosts
 ```
 
-* Edit /etc/kubernetes/config which will be the same on all hosts (master and node) to contain:
+* Edit /etc/kubernetes/config which will be the same on all hosts (master and node) to set
```

Comment:
change "will" to "should", add parentheses:
 "which will be the same on all hosts" -> "(which should be the same on all hosts)"